client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -375,7 +387,7 @@ export const useCommandsArrayWpcom = ( {
 		{
 			name: 'openActivityLog',
 			label: __( 'Open activity log' ),
-			callback: setStateCallback( 'openActivityLog' ),
+			callback: setStateCallback( 'openActivityLog', __( 'Select site to open activity log' ) ),
```

Comment:
Is Activity Log a product name, too, so it should use a title case? At least, this is what we do in the left menu.